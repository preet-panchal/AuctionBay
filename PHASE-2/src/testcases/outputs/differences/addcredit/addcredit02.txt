--- ./../testcases/outputs/expected/addcredit/addcredit02.txt	2023-02-03 00:25:47
+++ ./../testcases/outputs/actual/addcredit/addcredit02.txt	2023-03-06 15:53:07
@@ -1,28 +1,29 @@
-Welcome to AuctionBay!
---
-Here is a list of operations (what to input):
-- Login (login)
-- Create (create)
-- Delete (delete)
-- Advertise (advertise)
-- Bid (bid)
-- Refund (refund)
-- Add Credit (addcredit)
-- List All Items (listall)
-- Logout (logout)
-
-Enter operation:
-Enter username:
-Hello admin01 (AA)!
-
-Enter operation:
-Current credit balance for admin01: $10.00
-Enter the credit amount to add:
-Error: Invalid input for credit. Enter a number between 1-1000.
-Please try again.
-Enter the credit amount to add:
-Enter the username to add the credit to:
-New credit balance for admin01: $20.00
-
-Enter operation:
-admin01 successfully logged out.
\ No newline at end of file
+Welcome to AuctionBay!
+--
+Here is a list of operations (what to input):
+- Login (login)
+- Create (create)
+- Delete (delete)
+- Advertise (advertise)
+- Bid (bid)
+- Refund (refund)
+- Add Credit (addcredit)
+- List All Items (listall)
+- Logout (logout)
+
+Enter operation:
+Error: You must login first.
+
+Enter operation:
+Error: You must login first.
+
+Enter operation:
+Error: You must login first.
+
+Enter operation:
+Error: You must login first.
+
+Enter operation:
+Error: You must login first.
+
+Enter operation:
