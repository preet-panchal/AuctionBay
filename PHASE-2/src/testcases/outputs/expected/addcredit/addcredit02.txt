Welcome to AuctionBay!
--
Here is a list of operations (what to input):
- Login (login)
- Create (create)
- Delete (delete)
- Advertise (advertise)
- Bid (bid)
- Refund (refund)
- Add Credit (addcredit)
- List All Items (listall)
- Logout (logout)

Enter operation:
Enter username:
Hello admin01 (AA)!

Enter operation:
Current credit balance for admin01: $10.00
Enter the credit amount to add:
Error: Invalid input for credit. Enter a number between 1-1000.
Please try again.
Enter the credit amount to add:
Enter the username to add the credit to:
New credit balance for admin01: $20.00

Enter operation:
admin01 successfully logged out.